packages/console/src/onboarding/pages/SignInExperience/components/InspireMe/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { ConnectorType } from '@logto/connector-kit';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import Bulb from '@/assets/images/bulb.svg';
+import LightBulb from '@/assets/images/light-bulb.svg';
+import Button from '@/components/Button';
+import useConnectorGroups from '@/hooks/use-connector-groups';
+import type { OnboardingSieConfig } from '@/onboarding/types';
+
+import { randomSieConfigTemplate } from '../../sie-config-templates';
+import * as styles from './index.module.scss';
+
+type Props = {
+  onInspired: (template: OnboardingSieConfig) => void;
+};
+
+const InspireMe = ({ onInspired }: Props) => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const [isButtonHover, setIsButtonHover] = useState(false);
+  const BulbIcon = isButtonHover ? LightBulb : Bulb;
+  const [lastTemplateIndex, setLastTemplateIndex] = useState<number>();
+  const { data: connectorData, error } = useConnectorGroups();
+  const availableSocialTargets = error
+    ? []
+    : connectorData
+        ?.filter(({ type }) => type === ConnectorType.Social)
+        .map(({ target }) => target) ?? [];
+
+  const onEnterButton = () => {
+    setIsButtonHover(true);
+  };
+
+  const onLeaveButton = () => {
+    setIsButtonHover(false);
+  };
+
+  const handleInspire = () => {
```

Comment:
nit: If you do not intend to use the useCallback hook for the callback methods, there is no need to assign a new function variable to them. You can simply call the methods inline, which will have the same effect. If the callback methods are passed as props to child components, it may be beneficial to use useCallback to optimize performance.

Reply:
Cool, updated.